net/http/inc/THttpServer.h

```diff
@@ -62,6 +62,8 @@ class THttpServer : public TNamed {
 
    virtual void ProcessRequest(std::shared_ptr<THttpCallArg> arg);
 
+   virtual void ProcessBatchHolder(std::shared_ptr<THttpCallArg> arg);
```

Comment:
Do you need to copy the `shared_ptr`, or would a `const shared_ptr&` be enough?

Reply:
Same comment as before - just to be consistent with other methods of THttpServer